### PR TITLE
Calico network policy

### PR DESCRIPTION
**Pull request by bmckercher123**
- Added new Security article, Calico Network Policy.
- Updated Security.yml file to add it to left nav 

Please verify the following:
- Links work (and I used correct structure)
- Any markdown issues

Alex and Casey have reviewed the doc.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 7b0f8dc3a4d7438a0a2b9fc1c55a51923e1e5ca8

https://deploy-preview-2555--calico.netlify.com

File: master/security/calico-network-policy.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+The Kubernetes network policy specification defines the following behavior:
+- If no network policies apply to a pod, then all traffic to/from that pod is allowed.
+- If one or more network policies apply to a pod containing ingress rules, then only the ingress traffic specifically allowed by those policies is allowed, and all other traffic to/from the pod is denied.
+- If one or more network policies apply to a pod containing egress rules, then only the egress traffic specifically allowed by those policies is allowed, and all other traffic to/from the pod is denied.
+
+For compatibility with Kubernetes, **Calico network policy** follows the same behavior for Kubernetes pods.  For other endpoint types (VMs, host interfaces), Calico network policy is default deny. That is, only traffic specifically allowed by network policy is allowed, even if no network policies apply to the endpoint.
+
+### Before you begin...
+None
```
Comment (ozdanborne) [inline master/security/calico-network-policy.md]: I find these sections with `None` to be a bit jarring. Can we just remove the `Before you begin...` heading if there's nothing in it?

File: master/security/calico-network-policy.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+<pre>
+apiVersion: projectcalico.org/v3
+kind: <b>GlobalNetworkPolicy</b>
+metadata:
+  <b>name</b>: allow-tcp-port-6379
+</pre>
+{: .no-select-button}
+
+**Calico network and global policies** are applied using calicoctl. Syntax is similar to Kubernetes, but there a few differences. For help, see [calicoctl user reference]({{site.baseurl}}/{{page.version}}/reference/calicoctl/).
```
Comment (ozdanborne) [inline master/security/calico-network-policy.md]: The way this sentence is structured implies that it's **Calico global policy** but that's not a real thing!
```suggestion
**Calico network policies** and **Calico global network policies** are applied using calicoctl. Syntax is similar to Kubernetes, but there a few differences. For help, see [calicoctl user reference]({{site.baseurl}}/{{page.version}}/reference/calicoctl/).
```

File: master/security/calico-network-policy.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
+---
+title: Get started with calico network policy
+---
+
+### Big Picture
+Enforce the network traffic that is allowed or denied using rules in Calico network policy.
```
Comment (ozdanborne) [inline master/security/calico-network-policy.md]: This sentence feels a bit awkward to me. Perhaps this is all it needs?
```suggestion
Enforce which network traffic is allowed or denied using rules in Calico network policy. 
```

Comment (bmckercher123) [reply master/security/calico-network-policy.md]: That is good catch. This doc has been through multiple reviewers and edits so I appreciate that you are doing a final look.

File: master/security/calico-network-policy.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/calico-network-policy.md]: This is Alex decision for the template. His reasoning is that when/if we ever get the ability to allow customers to comment inline, this invites "why is this blank, or I think there are requirements before you start."  I see your point, but for now, this is the decision. The how to template is a work in progress so if we get other inputs we will change it.

File: master/security/calico-network-policy.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/calico-network-policy.md]: Yes, please change to: "Calico network policies and Calico global network policies are applied..."

Comment (bmckercher123) [reply master/security/calico-network-policy.md]: Fixed.

File: master/security/calico-network-policy.md (hunk 3 above)
Comment (bmckercher123) [reply master/security/calico-network-policy.md]: Fixed.

**Pull request merged**